src/datasets/summarizations/trend-regression.summarization.ts

```diff
@@ -32,7 +36,7 @@ export function queryFactory(points: TimeSeriesPoint[]) {
       -CHART_DIAGONAL_ANGLE * 5 / 8, -CHART_DIAGONAL_ANGLE / 2, -CHART_DIAGONAL_ANGLE / 4, -CHART_DIAGONAL_ANGLE / 8);
     const uQuicklyDecreasingLinearDynamic = trapmfR(-CHART_DIAGONAL_ANGLE * 5 / 8, -CHART_DIAGONAL_ANGLE / 2);
 
-    const uSmallRegressionError = trapmfR(0.75, 1.0);
+    const uSmallRegressionStd = trapmfR(0.09, 0.14);

```

Comment:
Could you link to your design doc or other documentation for these numbers? Help people understand why this is the best std deviation

Reply:
shame for me to say these are just random numbers... I just random picked some numbers and tested it with the data playground. They are definitely not the best and even have no evidence to support that they are good. I will not talk about the exact parameters I use in implementation in my design doc since the way I pick them is super stupid :(